commit a7c1a095c782fd5f177299e404c52c72fb9e4331
Author: Chao Peng <chao.p.peng@linux.intel.com>
Date:   Tue Apr 26 01:49:55 2016 -0400

    config: make PC board configurable
    
    User can select PC platform from full PC platform emulation (piix and q35)
    and light weight PC platform emulation. By default both are selected.
    
    Signed-off-by: Chao Peng <chao.p.peng@linux.intel.com>

diff --git a/default-configs/i386-softmmu.mak b/default-configs/i386-softmmu.mak
index b177e52..889b0c3 100644
--- a/default-configs/i386-softmmu.mak
+++ b/default-configs/i386-softmmu.mak
@@ -58,3 +58,5 @@ CONFIG_IOH3420=y
 CONFIG_I82801B11=y
 CONFIG_SMBIOS=y
 CONFIG_HYPERV_TESTDEV=$(CONFIG_KVM)
+CONFIG_PC_FULL=y
+CONFIG_PC_LITE=y
diff --git a/default-configs/x86_64-softmmu.mak b/default-configs/x86_64-softmmu.mak
index 6e3b312..99f5cc6 100644
--- a/default-configs/x86_64-softmmu.mak
+++ b/default-configs/x86_64-softmmu.mak
@@ -58,3 +58,5 @@ CONFIG_IOH3420=y
 CONFIG_I82801B11=y
 CONFIG_SMBIOS=y
 CONFIG_HYPERV_TESTDEV=$(CONFIG_KVM)
+CONFIG_PC_FULL=y
+CONFIG_PC_LITE=y
diff --git a/hw/i386/Makefile.objs b/hw/i386/Makefile.objs
index c1cc0e2..4c969d7 100644
--- a/hw/i386/Makefile.objs
+++ b/hw/i386/Makefile.objs
@@ -1,6 +1,8 @@
 obj-$(CONFIG_KVM) += kvm/
 obj-y += multiboot.o
-obj-y += pc.o pc_full.o pc_piix.o pc_q35.o pc_lite.o
+obj-y += pc.o
+obj-$(CONFIG_PC_FULL) += pc_full.o pc_piix.o pc_q35.o
+obj-$(CONFIG_PC_LITE) += pc_lite.o
 obj-y += pc_sysfw.o
 obj-y += intel_iommu.o
 obj-$(CONFIG_XEN) += ../xenpv/ xen/
